tests/test_main.py

```diff
@@ -1206,6 +1206,18 @@ def test_sort(dummy: Tensor) -> Tensor:
     return ep.sort(t)
 
 
+@compare_all
+def test_topk_0(dummy: Tensor) -> Tensor:
```

Comment:
Could you rename the tests to end with `_values` and `_indices` instead of `0` and `1`.